Cargo.toml

```diff
@@ -6,13 +6,17 @@ edition = "2018"
 
 [dependencies]
 tonic = { version="0.5.0", features=["prost"] }
-serde = { version="1.0.126", features=["derive"] }
+serde = { version="1.0.126", features=["derive"], optional = true }
 prost = "0.8.0"
 tokio = { version="1.8.0", features=["full"] }
+chrono = { version="0.4.19", optional = true }
 
 [build-dependencies]
 tonic-build = "0.5.0"
 anyhow = "1.0.41"
 
 [dev-dependencies]
 anyhow = "1.0.41"
+
+[features]
+chrono-integration = ["chrono"]
```

Comment:
Ideally we'd just use the automatic feature `chrono` unless you needed to activate other features or crates.